exercises/poker/src/example/java/Poker.java

```diff
@@ -0,0 +1,137 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class Poker {
+	private static class Card {
+		private int rank;
+		private int Suit;
+
+		public Card(String card) {
+			rank = Parserank(card);
+			Suit = ParseSuit(card);
+		}
+
+		public int getRank() {
+			return rank;
+		}
+	}
+
+	private static class Hand {
+		private String input;
+		private int score;
+
+		public Hand(String hand) {
+			this.input = hand;
+			this.score = ScoreHand(ParseCards(hand));
```

Comment:
Same goes for these two methods. `scoreHand` and `parseCards` are only used in this class therefore their implementations should be moved inside this class.